x-pack/test/functional/services/transform/wizard.ts

```diff
@@ -1085,12 +1118,15 @@ export function TransformWizardProvider({ getService, getPageObjects }: FtrProvi
       });
     },
 
-    async startTransform() {
+    // For continuous transform, no need to assert progress bar exists
```

Comment:
By default it will check for the progress bar unless specified otherwise. But I updated the comment to be more clear. [`be2e4e1` (#156128)](https://github.com/elastic/kibana/pull/156128/commits/be2e4e12a4760ece0733e15b113315b73b30d68c)

Reply:
Right, I've misread the comment. Thanks for clarifying it still!